app/routes.py

```diff
@@ -1 +1,248 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint,request,make_response, jsonify, abort
+from app import db
+from app.models.task import Task 
+from app.models.goal import Goal 
+from sqlalchemy import desc, asc 
+from datetime import date 
+import os
+from dotenv import load_dotenv
+import requests 
+load_dotenv()
+
+# CRUD for Tasks
+
+tasks_bp = Blueprint("tasks",__name__,url_prefix="/tasks")
+
+@tasks_bp.route("",methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+    try:
+        new_task = Task.from_dict(request_body)
+
+    except:
+        abort(make_response({
+            "details":"Invalid data"
+        },400))
+    
+    db.session.add(new_task)
+    db.session.commit() 
+    
+    response_body = {"task":new_task.to_dict()}
+    
+    return make_response(response_body, 201)
+
+"""
+Get all tasks - sort by descending order if users specify to do so, 
+otherwise sort by ascending order by default
+"""
+
+@tasks_bp.route("",methods=["GET"])
+def check_all_tasks():
+    order_query = request.args.get("sort")
+    
+    if order_query == "desc":
+        tasks = Task.query.order_by(desc("title"))
+    else:
+        tasks = Task.query.order_by(asc("title"))
```

Comment:
If the user does not specify "asc", we should do a regular `Task.query.all()` instead of defaulting to the `order_by(asc())` one.